packages/docs/src/components/DocsApi/ApiDocs.vue

```diff
@@ -25,7 +25,7 @@
                 <strong>{{ apiPropOption.name }}</strong>
               </td>
               <td>
-                <MarkdownView :value="$t(apiPropOption.description)" />
+                <MarkdownView :value="$tie(apiPropOption.description)" />
```

Comment:
There is no case where you want to pass string here directly, only the translation, no?

Reply:
I returned it as it was